spring-cloud-gcp-samples/spring-cloud-gcp-firestore-sample/README.adoc

```diff
@@ -0,0 +1,23 @@
+= Spring Cloud GCP Firestore Starter Example
+
+This code sample demonstrates how to read and write from Google Cloud Firestore with Spring Boot autoconfiguration from the Spring Cloud GCP Firestore starter module link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-firestore[Spring Cloud GCP Firestore Starter].
+
+== Running the example
+
+
+. https://firebase.google.com/docs/firestore/quickstart[Create a new Cloud Firestore database in your GCP project] if this has not already been done to allow Firestore to create and store entities.
+
+[start=2]
+. https://cloud.google.com/sdk/gcloud/reference/auth/application-default/login[If you are authenticated in the Cloud SDK], your credentials will be automatically found by the Spring
+Boot Starter for Google Cloud Firestore.
++
+Alternatively, https://console.cloud.google.com/iam-admin/serviceaccounts[create a service account from the Google Cloud Console] and download its private key.
++
+Then, uncomment the `spring.cloud.gcp.firestore.credentials.location` property in the link:src/main/resources/application.properties[application.properties] file and fill its value with the path to your service account private key on your local file system, prepended with `file:`.
+
+. Run the `FirestoreSampleApp` with
+```
+mvn spring-boot:run
+```
+
+This sample shows common operations of writing, reading, and deleting entities in Cloud Firestore.
```

Comment:
We probably don't need that, since it's a duplicate of the first sentence.

Reply:
done